2021-10-13-github-templates.md

```diff
@@ -0,0 +1,123 @@
+# OmbuLabs Standard Github Templates
+
+## Problem:
+
+New and junior contributors might be unsure about the information we require to
+efficiently review a PR. Similarly, it might be unclear what information is required when
+a new BUG or FEATURE is logged as an issue in Github.
+
+We have existing PR and ISSUE templates but they are inconsistent and might ask for
+unnecessary information from the contributor.
+
+The goal is to simplify  and streamline the steps required to make a contribution or log
+an issue.
+
+## Proposal:
+
+Come up with a consistent set of Github templates. The templates should include prompts
+only the ESSENTIAL information we require. We want to make sure that we don't ask for more
+information than we require - as this will discourages contributions.
+
+## Templates:
+
+### Pull Requests
+
+```
+    <!--- Before we start... -->
```

Comment:
We don't need this text

Reply:
I agree that we usually end up ignoring this section when craeting a PR, it gets too repetitive